rules-reviewed/jboss-eap/websphere/websphere-jms.windup.xml

```diff
@@ -0,0 +1,714 @@
+<?xml version="1.0"?>
+<ruleset xmlns="http://windup.jboss.org/schema/jboss-ruleset" id="websphere"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">>
+    <metadata>
+      <description>
+         This ruleset provides analysis of WebSphere proprietary classes and constructs that may require individual attention when migrating to JBoss EAP 6+.
+      </description>
+      <dependencies>
+         <addon id="org.jboss.windup.rules,windup-rules-javaee,2.2.0.Final"/>			
+         <addon id="org.jboss.windup.rules,windup-rules-java,2.2.0.Final"/>
+      </dependencies>
+      <sourceTechnology id="websphere" versionRange="(7,)"/>
+      <targetTechnology id="eap" versionRange="[6,)"/>
+   </metadata>
+    <rules>
+        <rule id="websphere-7Apvvd">
```

Comment:
these rule ids should follow our agreed convention, which is to use thousands:

-01000